docs/ballerina-by-example/examples/grpc-client-streaming/client/helloWorld.pb.bal

```diff
@@ -0,0 +1,78 @@
+// Copyright (c) 2018 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+package client;
+import ballerina.net.grpc;
+import ballerina.io;
+
+
+struct helloWorldStub {
+    grpc:Client clientEndpoint;
+    grpc:ServiceStub serviceStub;
+}
+function <helloWorldStub stub> initStub(grpc:Client clientEndpoint) {
+    grpc:ServiceStub navStub = {};
+    navStub.initStub(clientEndpoint, "non-blocking", descriptorKey, descriptorMap);
+    stub.serviceStub = navStub;
+}
+
```

Comment:
Please remove these unnecessary lines

Reply:
Fixed in fc9f60dd3cdaba97acb05287ea4d216112dbb632